integration-tests/funqy-amazon-lambda/src/test/java/io/quarkus/funqy/gcp/functions/test/GreetTestBase.java

```diff
@@ -24,7 +22,7 @@ public void testGreetNPE() {
         try {
             LambdaClient.invoke(Greeting.class, null);
```

Comment:
Why are you using the lambda client to test GCP?

Reply:
Leftover, I'll remove it.
There is no integration test support for the moment.

Reply:
In fact, it worse than that, IDEA refactor all the funqy tests when I refactory the one for GCF ... I'll revert this ...

Reply:
Change reverted.